src/map/lua/lua_baseentity.cpp

```diff
@@ -1659,6 +1659,21 @@ void CLuaBaseEntity::setStatus(uint8 status)
     m_PBaseEntity->updatemask |= UPDATE_HP;
 }
 
+/************************************************************************
+ *  Function: setStatus()
+ *  Purpose : Sends an 0x037 packet to player (m_PBaseEntity)
+ *  Example : pc:sendCharStatusPacket()
```

Comment:
I've never seen `pc:func` used, so this should be `player`. The Notes here should also mention: "We're doing this to line up with retail caps, but triggering this packet from Lua isn't ideal, etc." something in that realm